canonical-web-and-design/charmhub.io
# Migrate from LibSass to Dart Sass

## Done

- Migrate from LibSass to Dart Sass

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See everything works as expected


## Issue / Card

Fixes #450 

## Screenshots

[if relevant, include a screenshot]
